.github/workflows/deploy.yml

```diff
@@ -89,7 +89,7 @@ jobs:
           npm install -g purgecss
           purgecss -c purgecss.config.js
       - name: Deploy 🚀
-        if: github.event_name != 'pull_request'
```

Comment:
Ah yes. You're right it wasn't necessary.

This would've activated in non-PR branches, but because we don't test in those, the workflow never runs. So the effect is the same, but the new condition is a little more explicit.